packages/debug_page/lib/src/ui/details_screen/typography.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+
+class DebugPageTypography {
+  static TextStyle _getTextStyle(double fontSize) => TextStyle(
```

Comment:
I think having monospaced font for json etc. would be nice. Not in this PR just mentioning for the future